File: go/rna-transcription/rna_transcription.go
package strand

import "fmt"

func ToRNA(dna string) string {
	RNA := ""
    nucleotideMap := map[rune]string{'A': "U", 'C': "G", 'G': "C", 'T': "A"}
    for _, char := range dna {
        RNA = fmt.Sprintf("%s%s", RNA, nucleotideMap[char])
    }
	return RNA
}

File: javascript/rna-transcription/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const toRna = (strand) => {
  let nucleotideMap = {G: 'C', C: 'G', T: 'A', A: 'U'};
  return strand.split('').map(value => nucleotideMap[value]).join('');
};
